### PR TITLE
Add new translations in various places

### PR DESCRIPTION
**Pull request by pixeldesu**
Fixes up missing translations in places, moves old ones, etc.

15 strings left!

### CODE REVIEW

File: config/locales/views.en.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 en:
+  language: "English (US)"
```
Comment (nilsding) [inline config/locales/views.en.yml:2]: Disgusting.

File: app/views/user/_profile.haml [hunk 2]
```diff
@@ -13,28 +13,28 @@
       - if user.banned?
         %span.badge.badge-dark
           %i.fa.fa-fw.fa-ban
-          = t 'views.user.title.banned'
+          = t(".badge.banned")
```
Comment (raccube) [inline app/views/user/_profile.haml:16]: These badges are missing

**Review by raccube — CHANGES_REQUESTED** [review]

File: config/locales/views.en.yml [hunk 3]
```diff
@@ -511,3 +579,13 @@ en:
...
+    profile:
+      badge:
+        admin: "Admin"
+        banned: "Banned"
+        follows_you: "Follows you"
+        moderator: "Moderator"
+  question:
+    show:
+      questionbox:
+        title: "This question was not in your inbox? Answer it here!"
```
Comment (nilsding) [inline config/locales/views.en.yml]: Final newline missing

Comment (raccube) [reply config/locales/views.en.yml]: Also this string reads super non-English.

Comment (raccube) [inline config/locales/views.en.yml]:
```suggestion
        title: "If this question did not appear in your inbox, you can answer it here."
```

Comment (nilsding) [reply config/locales/views.en.yml]:
```suggestion
        title: "Want to answer this ask?  Question yourself!  :E"
```

File: config/locales/views.en.yml (hunk 1 above)
Comment (nilsding) [reply config/locales/views.en.yml:2]: still disgusting

**Review by nilsding — CHANGES_REQUESTED** [review]

**Review by raccube — CHANGES_REQUESTED** [review]

File: config/locales/views.en.yml (hunk 3 above)
Comment (raccube) [reply config/locales/views.en.yml]: forgot to submit this 🙃

Comment (raccube) [inline config/locales/views.en.yml]:
```suggestion
        title: "If this question did not appear in your inbox, you can answer it here."
```

**Review by raccube — CHANGES_REQUESTED** [review]

**Review by raccube — APPROVED** [review]

File: config/locales/voc.en.yml [hunk 4]
```diff
@@ -10,6 +10,7 @@ en:
     follow: "Follow"
     load: "Load more"
     login: "Sign in"
+    logout: "Logout"
```
Comment (raccube) [inline config/locales/voc.en.yml]:
```suggestion
    logout: "Sign out"
```

To match "Sign in"/"Sign up"

**Review by raccube — CHANGES_REQUESTED** [review]

**Review by raccube — APPROVED** [review]

**Review by raccube — APPROVED** [review]

**Comment by pixeldesu** [comment]
Can we merge this? I will address more of the Crowdin-related issues in another PR, this one is staying open for too long already

**Pull request merged**